docs/api.md

```diff
@@ -1951,6 +2008,39 @@ You can fully delete a link by issuing `DELETE` to its resource. This will remov
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+
+## Dataset Properties [/v1]
+
+_(introduced: version 2022.3)_
+
+Datasets are created and updated through Forms. Dataset-related Forms follow a new version of the [ODK XForms specification](https://getodk.github.io/xforms-spec) that allow them to define a Dataset and a mapping of Form Fields to Dataset Properties. Submissions from such a Form can create Entities within the Dataset defined in the Form. 
+
+Currently, Datasets and Dataset Properties are purely additive. Multiple Forms can add Properties to the same Dataset and multiple Forms can create Entities in the same Dataset. Not all Properties of a Dataset have to be included in a Form for that Dataset. For example, one Form publishing to a Dataset called `trees` could add `location` and `species`, while another could add `species` and `circumference`. The Properties of the Dataset would be the union of Properties from all Forms for that Dataset (`location`, `species`, `circumference`).
+
+Another way to think of this is how Submissions/Entities from a particular Form will affect a Dataset; which Properties will be saved on an Entity from a given Form Submission. Note that it is not necessary that a Form will save to all Properties of a Dataset, so the endpoint also returns a `inForm` flag for each property which is true only if the Form affects that Property.
```

Comment:
I think the ; should be a colon? The second clause is not independent. I'm not totally sure I understand the intent of this sentence, though. Maybe it could be reworked or removed? I find the paragraph above clearer.